src/main/java/org/openmaptiles/layers/Transportation.java

```diff
@@ -320,6 +384,43 @@ List<RouteRelation> getRouteRelations(Tables.OsmHighwayLinestring element) {
           }
         }
       }
+      // Similarly Ireland.
+      refMatcher = IRELAND_REF_NETWORK_PATTERN.matcher(ref);
+      if (refMatcher.find()) {
+        if (ireland == null) {
+          if (!loggedNoIreland.get() && loggedNoIreland.compareAndSet(false, true)) {
+            LOGGER.warn("No IE polygon for inferring route network types");
+          }
+        } else {
+          try {
+            Geometry wayGeometry = element.source().worldGeometry();
+            if (ireland.intersects(wayGeometry)) {
+              Transportation.RouteNetwork networkType;
+              String network;
+              String highway = coalesce(element.highway(), "");
+              switch (highway) {
+                case "motorway" -> {
+                  networkType = Transportation.RouteNetwork.IE_MOTORWAY;
+                  network = "omt-ie-motorway";
+                }
+                case "trunk", "primary" -> {
+                  networkType = RouteNetwork.IE_NATIONAL;
+                  network = "omt-ie-national";
+                }
+                default -> {
+                  networkType = RouteNetwork.IE_REGIONAL;
+                  network = "omt-ie-regional";
+                }
+              }
```

Comment:
Could we add these strings like `"omt-ie-motorway"` as an optional argument to RouteNetwork enum, so we could just do:

```java
Transportation.RouteNetwork networkType = switch(highway) {
  case "motorway" -> RouteNetwork.IE_MOTORWAY;
  case "trunk", "primary" ->  RouteNetwork.IE_NATIONAL;
  default -> RouteNetwork.IE_REGIONAL;
};
result.add(new RouteRelation(refMatcher.group(), networkType.network, networkType, (byte) -1, 0));
```

And same or GB?